diff/testdata/gen.sh

```diff
@@ -11,3 +11,4 @@ git diff --no-index /dev/null "empty space.txt" > empty_space.diff
 git diff --no-index golint.{old,new}.go > golint.diff
 git diff --no-index empty.txt /dev/null > newline_and_empty_deleted.diff
 git diff --no-index golint.{old,new}.go >> newline_and_empty_deleted.diff
+git diff --no-index \"日本語\".{old,new}.txt >> \"日本語\".diff
```

Comment:
This quote should not be necessary (**"**日本語**"**) here right?. (It can be just 日本語.old.txt)
If it intends to test file name which contains `"`, I prefer testing these cases with unit tests.

Reply:
That's right.
I moved it to the unit test. https://github.com/haya14busa/reviewdog/pull/183/commits/13bd203efe436d8bf3ddd6ec8f877ae78dbd202a